src/data/events/events.js

```diff
@@ -77,14 +77,15 @@ const events = [
 		date: getDateTime(2022, 11, 9, 18),
```

Comment:
Also i don't believe we are doing this time and location anymore, so try to change it to TBD or comment it out